apps/web-embed/pages/PageWebEmbedPrimePurchase.tsx

```
@@ -0,0 +1,355 @@
+        const pricePerMonth = periodDuration === 'P1Y' ? price / 12 : price;
+        const showSaveBadge = periodDuration === 'P1Y';
+
+        return (
+          <div
+            key={p.identifier}
+            style={itemStyle(selected)}
+            onClick={() => setSelectedPackageId(p.identifier)}
+            onKeyDown={() => {
+              // noop
+            }}
+            role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)